packages/admin/src/DirtyHandler.tsx

```diff
@@ -91,8 +91,14 @@ class DirtyHandlerComponent extends React.Component<IProps & WrappedComponentPro
 
     private submitBindings() {
         return Promise.all(
-            this.bindings.map((binding) => {
-                return binding.binding.submit();
+            this.bindings.map(async (binding) => {
+                let submitResult = await binding.binding.submit();
+
+                if(!submitResult) {
+                    submitResult = {}
```

Comment:
```suggestion
                    submitResult = {};
```

prettier is not executed here?

Reply:
For some reason PHPStorm stopped fixing prettier and eslint issues and the linting in the precommit-hook still didn't fail